src/main/java/net/pms/formats/OGG.java

```diff
@@ -0,0 +1,27 @@
+package net.pms.formats;
+
+public class OGG extends Format {
+
+	@Override
+	public Identifier getIdentifier() {
+		return Identifier.OGG;
+	}
+
+	@Override
+	public boolean transcodable() {
+		return true;
+	}
+
+	public OGG() {
+		type = VIDEO;
+	}
+
+	@Override
+	public String[] getSupportedExtensions() {
+		return new String[] {
+			"ogg",
+			"ogm",
+			"ogv"
```

Comment:
@Sami32 Can't you explain properly what you mean instead of just giving small pieces of information that I don't get?